### PR TITLE
VMO-5463 fix 'undefined mapping' issue from QA testing

### PR DESCRIPTION
**Pull request by bulatgab**

### CODE REVIEW

File: src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts [hunk 1]
```diff
@@ -5,7 +5,12 @@ import {IdGeneratorUuidV4} from '@floip/flow-runner/dist/domain/IdGeneratorUuidV
 import {ISelectOneResponseBlock} from '@floip/flow-runner/dist/model/block/ISelectOneResponseBlock'
 import Vue from 'vue'
 import {cloneDeep, find, reject} from 'lodash'
-import BaseStore, {actions as baseActions, mutations as baseMutations, getters as baseGetters, IEmptyState} from '@/store/flow/block-types/BaseBlock'
+import BaseStore, {
+  actions as baseActions,
+  mutations as baseMutations,
+  getters as baseGetters,
+  IEmptyState,
+} from '@/store/flow/block-types/BaseBlock'
```
Comment (bulatgab) [inline src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts:13]: unrelated change (fix eslint long line)

**Review by safydy-r — APPROVED** [review]
- [x] don't forget to build, and update `yarn.lock` in voto5
